content/visit-belgium-tips.adoc

```diff
@@ -0,0 +1,71 @@
+
+== To See or Visit
+
+=== Antwerp
+
+==== Zoo of Antwerp
+
+The [Zoo of Antwerp](https://www.zooantwerpen.be/en/) is the oldest animal park in the country and one of the oldest in the world, established on 21 July 1843. It's located in the city center, next to the central station.
+
+==== Flandria Boat / Waterbus
+
+*A [Flandria boat tour](https://flandria.nu/en/homepage-en/) is the perfect way to discover Antwerp and its world-class port from a different angle.*
+
+[DeWaterbus](https://www.dewaterbus.be/en) is a cheaper alternative as it's more a public transport, that connects the center of Antwerp with the port and neighbouring cities like Kruibeke and Hemiksem.
+
+==== Brewery De Koninck
+
+*The most fun adventure in Antwerp? A [tour through our City Brewery](https://www.dekoninck.be/en/interactive-tour)! That is not only a must-see, but especially a must-do due to the countless interactive experiences.*
+
+==== The Playground
+
+For boardgame lovers! *Are you up for some games? Browse our collection of more than 400 games that you can play right here in [The Playground](https://www.facebook.com/theplaygroundx/). As a board game café and shop, our aim is to provide you with plenty of games to play, quality coffee, Belgian beers and more.*
+
+==== Red Star Line Museum
+
+*Two million passengers made the crossing from Antwerp to North America aboard the steamships of the Red Star Line. Their story, that of the shipping company that transported them and that of the city and the port from which they departed, are central to the [Red Star Line Museum](https://redstarline.be/en)’s main exhibition.*
+
+==== MAS
+
+*The [MAS aims to be an international and pioneering museum](https://mas.be/) about the global connectedness between people and collections from different cultures and environments. With a river of tales, viewed from multiple perspectives, we engage everyone to be curious about the present, past and future of Antwerp and the world.*
+
+==== Plantin-Moretus
+
+*In 1576, Plantin relocated his printing works to the Vrijdagmarkt. His family lived and worked there for three hundred years. They converted the ‘Gulden Passer’ (‘Golden Compasses’) into a beautiful mansion. The Moretus family cherished their [printing works, which had become a part of Antwerp’s heritage](https://museumplantinmoretus.be/en).*
+
+=== Across Belgium
+
+==== Brugge and Gent
+
+The two cities [Brugge](https://www.visitbruges.be/en) and [Gent](https://visit.gent.be/en/see-do/inquiry-desk-visit-gent) are competing as they have the same values to offer: water and boats in the city, shopping streets, historical buildings,... Maybe best to visit both so you can make an informed choice for a next return visit?
+
+==== Pairi Daiza
+
+As "Probably The Best Zoo in Europe," [Pairi Daiza](https://www.pairidaiza.eu) is a must-see for animal lovers. It's on the other side of Belgium if you are in Antwerp, but hey, it's a tiny country ;-)
```

Comment:
Maybe better to say 100km for those with a bit of a grander idea of country sizes?

Reply:
good remark, done 
BTW 102km according to maps ;-)